Test/test/main/CalculatorDateAndTimeTest.java

```diff
@@ -0,0 +1,113 @@
+package main;
+import org.junit.Before;
+import org.junit.Test;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.TimeZone;
+
+import static org.junit.Assert.*;
+
+public class CalculatorDateAndTimeTest {
+
+    private ZonedDateTime testDate;
+    @Before
+    public void resetDate() {
+        testDate = ZonedDateTime.of(2018, 6, 26, 0, 0, 0, 0, ZoneId.of(TimeZone.getDefault().getID()));
+    }
+    @Test
+    public void toStringTest() {
+        String actual = CalculatorDateAndTime.toString(testDate);
+        String expected = "2018-06-26";
+        assertEquals(expected, actual);
+    }
+    @Test
+    public void calculateAddYearMonthDay() {
+        testDate = CalculatorDateAndTime.calculate(testDate, "+1y +1m +1d");
+        String actual = testDate.format(DateTimeFormatter.ISO_LOCAL_DATE);
```

Comment:
no need to overwrite testDate you can
String actual = CalculatorDateAndTime.calculate(testDate, "+1y +1m +1d")
.format(DateTimeFormatter.ISO_LOCAL_DATE);

Reply:
apply everywhere